Makefile

```diff
@@ -2,11 +2,13 @@ ifeq ($(findstring clean,$(MAKECMDGOALS)),)
 -include Makefile.config
 endif
 
+# TODO: Replace --profile=$(DUNE_PROFILE) by --release when we require on dune >= 2.5
+
```

Comment:
This would need to be slightly more complicated - the idea was that you can say `make DUNE_PROFILE=dev`

Reply:
we could still have something like that:
```
ifeq ($(DUNE_PROFILE),release)
DUNE_PROFILE_ARG=--release
else
DUNE_PROFILE_ARG=--profile=$(DUNE_PROFILE)
endif

Reply:
Indeed, that was my slightly more complicated 🙂